src/app/frontend/deploy/createnamespace.html

```diff
@@ -16,12 +16,12 @@
 
 <md-dialog aria-label="Create a new namespace" layout="column" layout-padding>
   <md-content layout-padding>
```

Comment:
Btw, I did a little research and it happens that we're using the md-dialog directive in a wrong way. Take a look at the documentation: https://material.angularjs.org/latest/api/service/$mdDialog

Basically, there should be md-dialog-content here, not md-content. Also, when you remove layout-padding from line 17 and 18 and remove the CSS styles you've added in this CL, everything starts looking normal.

Please test and tell me what you think :)